opal-shell/src/main/java/org/obiba/opal/shell/commands/AnalyseCommand.java

```diff
@@ -115,6 +122,47 @@ public int execute() {
     return 0;
   }
 
+  private class AnalysisValueTableResolver {
+    ValueTable view = null;
+
+    ValueTable resolve(ValueTable source, String variableNamesCsv) {
+      List<String> variableNames = getVariableNames(variableNamesCsv);
+      if (variableNames.isEmpty()) return source;
+
+      view = createView(source, variableNames);
+      log.debug("Created view {} for variables {}", view.getName(), variableNamesCsv);
+
+      return view;
+    }
+
+    private ValueTable createView(ValueTable table, List<String> variableNames) {
+      String viewName = table.getName() + "View";
+      View view =  View.Builder
+        .newView(viewName, table)
+        .select(variable -> variableNames.contains(variable.getName()))
+        .build();
+
+      viewManager.addView(table.getDatasource().getName(), view, null, null);
```

Comment:
Is it necessary?

Reply:
No! I thought it was querying Magma.

Reply:
Then remove these ViewManager references